packages/assets-controllers/src/DeFiPositionsController/fetch-positions.test.ts

```diff
@@ -0,0 +1,75 @@
+import { fetchPositions } from './fetch-positions';
+
+describe('fetchPositions', () => {
+  const mockAccountAddress = '0x1234567890123456789012345678901234567890';
+  const mockApiUrl =
+    'https://defi-services.metamask-institutional.io/defi-data/positions';
+
+  /**
+   *
+   * @param status - The status code to return
+   * @param jsonResponse - The JSON response to return
+   */
+  function mockNextFetchResponse(status: number, jsonResponse?: unknown) {
+    const mockFetch = jest.spyOn(global, 'fetch');
+    mockFetch.mockResolvedValueOnce({
+      status,
+      ...(Boolean(jsonResponse) && {
+        json: () => Promise.resolve(jsonResponse),
+      }),
+    } as unknown as Response);
```

Comment:
Thoughts on using `nock` instead of mocking the fetch calls?

Reply:
Good call. Changing it to use nock now.